deps/chakrashim/src/v8booleanobject.cc

```diff
@@ -23,10 +23,12 @@
 namespace v8 {
 
 using jsrt::ContextShim;
+using jsrt::IsolateShim;
 
-Local<Value> BooleanObject::New(bool value) {
```

Comment:
Wonder we should still keep deprecated implementation, in case anybody using deprecated API? It can be removed when the declaration is removed?